deps/ncrypto/ncrypto.cc

```diff
@@ -4666,6 +4666,9 @@ std::pair<std::string, std::string> X509Name::Iterator::operator*() const {
   unsigned char* value_str;
   int value_str_size = ASN1_STRING_to_UTF8(&value_str, value);
 
+  std::string out(reinterpret_cast<const char*>(value_str), value_str_size);
+  OPENSSL_free(value_str);  // free after copy
+
   return {
       std::move(name_str),
       std::string(reinterpret_cast<const char*>(value_str), value_str_size)};
```

Comment:
This change was missed; otherwise this is a use-after-free

```suggestion
      std::move(out)
```